README.md

```diff
@@ -36,15 +38,126 @@ characterize the data they represent. Some properties are categorized as `requir
 If a submission lacks a required property, it cannot be accepted. Preferred properties can denote
 two things: the property is being highlighted as it has become more desired by the community or
 the property is being promoted to required. All properties not designated either `required` or
-`preferred` are still sought by GDC, but submissions without them are allowed. 
+`preferred` are still sought by GDC, but submissions without them are allowed.
 
 The properties have further validation through their entries. Legal values are defined in each
 property. For the most part these are represented in the `enum` categories although some keys,
 such as `submitter_id`, will allow any string value as a valid entry. Other numeric properties
 can have maximum and minimum values to limit valid entries.  For examples of what a valid entry
-would look like, each node has a mock submission located in the `examples/valid/` directory. 
+would look like, each node has a mock submission located in the `examples/valid/` directory.
+
+## Dictionary Changes
+
+The following is an attempt to layout guidelines for the level of
+impact of changes to the dictionary by categorizing them into
+**Breaking Changes**, **Entity Relation Additions**, **Schema Additions**,
+**Cosmetic Corrections**.
+
+### Breaking Changes
+
+Breaking changes are changes to the dictionary such that previously
+allowable data is invalid against the new schema, e.g. a **removal** of
+part of the dictionary.
+
+N.B. That not all changes classified here as Breaking Changes are
+promised to require a data migration.  It is possible that no data
+exists in the GDC that is invalidated by the change, e.g. making a
+field required that has never been left blank.  This should be
+confirmed against the corpus of data and the userbase should be
+notified of a break in backwards-compatibility.
+
+**Breaking Changes include**:
+- Renaming/removing anything that is not a description or comment
+  - Removing an entity schema
+  - Removing a property's allowed `type`
```

Comment:
What happens if we change a string or number to an enum but capture the values that exist in PROD?